newsfragments/461.feature.rst

```diff
@@ -0,0 +1,3 @@
+Add :func:`trio._util.current_thread` as a safer alternative to
```

Comment:
`trio._util.main_thread` isn't a public function that we'll document, so it shouldn't be mentioned in the end-user release notes. Here we should just be saying that KeyboardInterrupt handling is now robust against third-party libraries that break `threading.current_thread`. (And I guess that makes this a `bugfix`, not a `feature`?)